dockerfiles/launcher/launcher_funcs.sh

```diff
@@ -166,6 +166,22 @@ docker_run_with_debug() {
   fi
 }
 
+docker_run_with_volumes_lib() {
+  if has_lib_volume; then
+    docker_run_with_debug -v "${CHE_LIB_VOLUME}":/data/lib "$@"
```

Comment:
@sbryzak - Hey Shane, any thoughts on the modifications required?  If it would help, i can do a screen share for 45 minutes any time this week to guide you through what we should do.  It would not take long, I think.

I'd like to add the M9 label for this issue, but cannot do it unless we agree on these additional changes, which are needed.